### PR TITLE
add rayonism experimental flag, add consensus json-rpc namespace

### PR DESCRIPTION
**Pull request by garyschulte**
## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.hyperledger.besu.config.experimental;
+
+import picocli.CommandLine.Option;
+
+/** Experimental support for eth1/2 merge, rayonism hackathon: https://rayonism.io/ */
+public class RayonismMerge {
```
Comment (matkt) [inline config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java:6]: IMO we need to change the name . RayonismMergeOptions to do like other option classes

File: config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.hyperledger.besu.config.experimental;
```
Comment (matkt) [inline config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java:1]: this does not seem to me the right package. unstable options are here `org.hyperledger.besu.cli.options.unstable`

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 3]
```diff
@@ -1203,6 +1204,7 @@ void setBesuConfiguration(final BesuConfiguration pluginCommonConfiguration) {
   private void enableExperimentalEIPs() {
     // Usage of static command line flags is strictly reserved for experimental EIPs
     commandLine.addMixin("experimentalEIPs", ExperimentalEIPs.class);
+    commandLine.addMixin("experimentalMerge", RayonismMerge.class);
```
Comment (matkt) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:1207]: maybe I'm wrong but for me it's not an EIP but a feature. I would have added it in this part

https://github.com/hyperledger/besu/blob/56255c11b3f5757f42f1d8bf0c62d786b54afdfb/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L1264

File: config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java (hunk 1 above)
Comment (matkt) [inline config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java:6]: Maybe Import net.consensys.quorum.mainnet.launcher.options.Options;

You can look at the others options classes like LauncherOptions class

**Review by matkt — COMMENTED** [review]
Some comments to discuss

File: config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java (hunk 1 above)
Comment (garyschulte) [reply config/src/main/java/org/hyperledger/besu/config/experimental/RayonismMerge.java:6]: cool.  yeah I tried a few different names.  I decided to just go with something and change it later 👍

**Pull request merged**